cdap-data-fabric/src/main/java/co/cask/cdap/data2/nosql/NoSqlTransactionRunner.java

```diff
@@ -159,4 +159,8 @@ public void discardDataset(Dataset dataset) {
       // no-op
     }
   }
+
+  interface TableDatasetAccesor {
```

Comment:
Typo, `Accessor`?

Reply:
Also, better call it `TableDatasetSupplier`

Reply:
Done